cmd/git_enable.go

```diff
@@ -0,0 +1,60 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"net/http"
+	"os"
+	"qovery.go/io"
+	"strings"
+)
+
+var gitEnableCmd = &cobra.Command{
+	Use:   "enable",
+	Short: "Enables git - Qovery webhooks in given project",
+	Long: `Enables git - Qovery webhooks in given project e.g.
+qovery git enable https://gitlab.com/pjeziorowski/publicproject
+enables sending notifications about events in https://gitlab.com/pjeziorowski/publicproject project`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) == 0 {
+			println("Usage: qovery git enable <REPO_URL>")
+			os.Exit(1)
+		}
+		group, projectName := sanitize(args[0])
+		enableWebhooks(group, projectName)
+	},
+}
+
+func sanitize(repoUrl string) (group string, projectName string) {
+	repoWithoutPrefix := strings.ReplaceAll(repoUrl, "https://gitlab.com/", "")
+	repoWithoutPrefixAndSuffix := strings.ReplaceAll(repoWithoutPrefix, ".git", "")
+	split := strings.Split(repoWithoutPrefixAndSuffix, "/")
+
+	if len(split) != 2 {
+		println("Usage: qovery git enable <REPO_URL>")
+		println("where <REPO_URL> is URL to your git project e.g.")
+		println("https://gitlab.com/pjeziorowski/publicproject")
```

Comment:
please replace your username with qovery and may create a real repo to be forked